_includes/contributor_side_nav.html

```diff
@@ -0,0 +1,12 @@
+<div class="col-md-3 get-started-cloud-sidebar">
+  <div class="sticky-top">
+    <ul id="get-started-contributor-sidebar-list">
+      <li>
+        <a href="#pytorch-1">Issue 1 March 2021</a>
```

Comment:
With this approach, it looks like we'll have to hardcode the date each time they add a new issue. Is it possible to add date front matter to the markdown file, pass it to this includes, and then render that date here?

Reply:
Yes, it is possible. That approach was initially recommended but to meet the deadline it was agreed to just hard code it for now. This will need to be changed around the same time the looping for the issues is implemented